cy/cypress/e2e/1-baseline/file_upload.cy.js

```diff
@@ -3,78 +3,77 @@ import {randString} from "../../support/commands";
-        cy.drupalLogin(testUser, 'password')
-        cy.visit('media/add/video')
-        cy.get('[data-drupal-selector="edit-name-0-value"]').type(`${testFileName}`)
-        cy.get('[data-drupal-selector="edit-field-media-video-file-0-upload"]').selectFile('cypress/fixtures/media/video_test.mp4')
-        cy.wait(500)
-        cy.confirm()
-        cy.execDrush(`sql:query 'SELECT * FROM media_field_data' | grep video | grep ${testFileName}`)
+        uploadFileAndConfirm('video', testFileName, 'video_test.mp4');
     })
 
     it('Test image file upload', () => {
-        cy.drupalLogin(testUser, 'password')
-        cy.visit('media/add/image')
-        cy.get('[data-drupal-selector="edit-name-0-value"]').type(`${testFileName}`)
-        cy.get('[data-drupal-selector="edit-field-media-image-0-upload"]').selectFile('cypress/fixtures/media/image_test.jpeg')
-        cy.wait(500)
-        cy.get('[data-drupal-selector="edit-field-media-image-0-alt"]').type('blah blah')
-        cy.confirm()
-        cy.execDrush(`sql:query 'SELECT * FROM media_field_data' | grep image | grep ${testFileName}`)
+        uploadFileAndConfirm('image', testFileName, 'image_test.jpeg', 'blah blah');
     })
 
     it('Test PDF file upload', () => {
-        cy.drupalLogin(testUser, 'password')
-        cy.visit('media/add/document')
-        cy.get('[data-drupal-selector="edit-name-0-value"]').type(`pdf${testFileName}`)
-        cy.get('[data-drupal-selector="edit-field-media-document-0-upload"]').selectFile('cypress/fixtures/media/pdf_test.pdf')
-        cy.wait(500)
-        cy.confirm()
-        cy.execDrush(`sql:query 'SELECT * FROM media_field_data' | grep document | grep pdf${testFileName}`)
+        uploadFileAndConfirm('document', `pdf${testFileName}`, 'pdf_test.pdf');
     })
 
     it('Test DOCX file upload', () => {
-        cy.drupalLogin(testUser, 'password')
-        cy.visit('media/add/document')
-        cy.get('[data-drupal-selector="edit-name-0-value"]').type(`docx${testFileName}`)
-        cy.get('[data-drupal-selector="edit-field-media-document-0-upload"]').selectFile('cypress/fixtures/media/docx_test.docx')
-        cy.wait(500)
-        cy.confirm()
-        cy.execDrush(`sql:query 'SELECT * FROM media_field_data' | grep document | grep docx${testFileName}`)
+        uploadFileAndConfirm('document', `docx${testFileName}`, 'docx_test.docx');
     })
 
     after('Clean up', () => {
         // Remove all test files
-        cy.drupalLogin()
-        cy.visit('admin/content/media')
-        cy.get('.select-all > .form-checkbox').check()
-        cy.get('#edit-action').select('media_delete_action')
-        cy.get('#edit-submit').click()
-        cy.get('#edit-also-delete-file').check()
-        cy.confirm()
+        cy.removeTestFiles();
+        
         // Delete test user
         cy.execDrush(`user:cancel -y --delete-content ${testUser}`)
     })
 
-
 })
+
+
+function uploadFileAndConfirm(type, fileName, filePath, altText = '') {
```

Comment:
I like this, however I think it would be better to separate the arrangement part of the test from the assertion part of the test. Currently, I think it is too hidden what is being asserted. I think it would be better to split this function into two: `uploadFile()` and `confirmFileUploadedSuccessfully()`. 

While `confirmFileUploaded()` will only be one line (`cy.execDrush(...)`), I think it is still worth it as it greatly improves readability